apps/src/code-studio/components/libraries/LibraryClientApi.js

```diff
@@ -25,6 +26,22 @@ export default class LibraryClientApi {
     );
   }
 
+  unpublish(project, callback) {
```

Comment:
It's a bit confusing as to when to use `unpublish` vs `delete`. Would it make sense to make `unpublish` the default action, have it check whether we are in an app, and then take actions accordingly? I'm thinking about here: https://github.com/code-dot-org/code-dot-org/blob/c09f509ce6d5ca5af6effb6023066374d091c95c/apps/src/code-studio/components/libraries/LibraryPublisher.jsx#L234-L246

Another possibility is to add some comments to make it clear when to use each.

Reply:
all good! i'll go ahead and update `<LibraryPublisher/>` now so i don't forget 😅

Reply:
Thanks for making this change!